internal/pkg/addons/output.go

```diff
@@ -0,0 +1,23 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package addons
+
+// Output represents an output from a CloudFormation template.
+type Output struct {
+	// Name is the Logical ID of the output.
+	Name string
+
+	isSecret        bool
+	isManagedPolicy bool
+}
+
+// IsSecret returns true if the output value refers to a SecretsManager ARN. Otherwise, returns false.
```

Comment:
Is this preference or any special reason to use this getter instead of making `isSecret` public?

Reply:
I have no idea why I did this :P I'll change it to having those fields as public and getting rid of these methods.